docs/book/src/topics/federated-identity-credential.md

```diff
@@ -44,11 +44,10 @@ azwi serviceaccount delete phase federated-identity \
 To create a federated identity credential, login to [Azure Cloud Shell][1] and run the following commands:
 
 ```bash
-# Get the client and object ID of the AAD application
-export APPLICATION_CLIENT_ID="$(az ad sp list --display-name "${APPLICATION_NAME}" --query '[0].appId' -otsv)"
-export APPLICATION_OBJECT_ID="$(az ad app show --id "${APPLICATION_CLIENT_ID}" --query id -otsv)"
+# Get the object ID of the AAD application
```

Comment:
In the docs we show how to set up a federated identity credential using:
1. `az rest` in cloud shell
2. `azwi` CLI
3. Azure portal UI

Could we add a 4th option with just azure cli and add these in that section? That way the user can choose what option works best for them. 

nit: do we need cloud shell for these commands?

Reply:
cloud shell is updated with latest az cli (.39 now) so we shouldn't need `az rest` anymore i think

Reply:
Yeah I would think we would want to drop the `az rest` method at this point right? I'm guessing cloud shell was suggested because it provides an up-to-date environment. I can add a note there to use cloud shell or a local terminal